hmt_escrow/job.py

```diff
@@ -838,12 +836,33 @@ def final_results(self, priv_key: bytes, gas: int = GAS_LIMIT) -> Dict:
             bool: returns True if IPFS download with the private key succeeds.
 
         """
-        final_results_url = self.job_contract.functions.getFinalResultsUrl(
+        reputation_oracle_results_url = self.job_contract.functions.getReputationOracleResultsUrl(
         ).call({
             'from': self.gas_payer,
             'gas': gas
         })
-        return download(final_results_url, priv_key)
+        return download(reputation_oracle_results_url, priv_key)
+
+    def getIpnsUrl(self, keyName: str) -> str:
+        return getIpnsLink(keyName)
+
+    def launcher(self, escrow_contract: Contract, gas_payer: str,
```

Comment:
Ah, saw you moved it here. Any particular reason why?

Reply:
No reason. I'll restore the order of functions from before.

Reply:
The order doesn't matter much, just harder to see / not as obvious in a diff view when reviewing

Reply:
Ok, it has been restored. If there was a reason i don't remember, but it makes sense for it to be up there.